1bash.template

```diff
@@ -451,7 +453,9 @@ CLAYMORE_OPTS=""                                 # Claymore optional arguments.
 EQUIHASH_MINER="DSTM"                            # Choose "DSTM" or "EWBF" or "BMINER"
 EQUIHASH_WALLET_FORMAT="$WALLET_ADDRESS_FORMAT"  # Chose the format your pool requires 
 DSTM_OPTS=""                                     # DSTM Miner optional arguments add "--telemetry=$IPW:42000" if you want web info
-EWBF_VERSION="3_4"                               # choose 3_3  or  3_4 # EWBF ZEC Miner version
+EWBF_VERSION="3_4"                               # choose 3_3  or  3_4 or new v0_3# EWBF ZEC Miner version
```

Comment:
I wonder if it would be better to explain that the version should match the name of the miner folder. And maybe it is better to distinguish EWBF3 and EWBF0 since in case someone would like to use EWBF3 for Zcash and EWBF0 for Bitcoin Gold the current solution doesn't allow such an option.

Reply:
ewbf version naming is so confusing 
Older versions are 0.3.3b and 0.3.4b, and ...,New one is v0.3 
The new version should be capable of mining older algorithms.
With EWBF0 and EWBF3  where do we assign which coin to be mine with which miner?
We only have equihash_miner variable, then we need a EWBF0 coins and EWBF3 coins and so many new variables in 0miner...

Reply:
Sadly the new version has no support for the old algo… shame on ewbf for choosing the most horrible naming and verioning scheme...

Reply:
lol ... No idea whats with all the 0.3 versioning schemes.
Problem only occur if user wants to auto switch multiple new and old equihash algos with wtm.
Lets hope dstm gives a general update for both.

Reply:
ok, so just change that comment removing any explict reference to ZEC, and add a whitespace between 'v0.3' and '#', just in case, I don't want to see someone using 'v0.3#' as the version 😅